.pre-commit-config.yaml

```
@@ -25,6 +25,12 @@ repos:
         language: system # Using system as entry is now a direct path
         types: [python]
 
+    -   id: pyright
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)